taskcluster/ci/beetmover/kind.yml

```diff
@@ -48,3 +49,50 @@ jobs:
             kind: build
             tier: 1
             platform: windows/x86_64
+    addons-bundle:
+        worker-type: beetmover
+        worker:
+            chain-of-trust: true
+            max-run-time: 3600
+        run-on-tasks-for: [action:release-promotion]
```

Comment:
I think it's fine to leave this as it is informative, but just fyi it is a no-op since the `ship-addons` and `promote-addons` target_task_methods do not apply the `run-on-tasks-for` filter.